### PR TITLE
style: edit styles (fixes #5626, fixes #5625, fixes #5622, fixes #5629)

### LINKED ISSUE #5622: Darkt Theme, prox port dropdown has wrong color
##### Brief Description

Commit hash:  ec500b667306e913cd37d14b6aa1ef8bddfa4d91

Reproducible: Always

##### Steps to reproduce

1. Go to Settings/Advanced

##### Observed Behavior

Screenshot_20190416_001152

##### Expected Behavior

The dropdown should also have the dark theme.

### LINKED ISSUE #5625: Dark theme friend info pane is difficult to read
##### Brief Description

Commit hash:  ec500b667306e913cd37d14b6aa1ef8bddfa4d91

Reproducible: Always

##### Steps to reproduce

1. Right click friend in friend list.
2. Click show details

##### Observed Behavior
Multiple text fields of the friend information are very difficult to read.
image

##### Expected Behavior
Correctly coloured text/background so that all info is easily legible.

### LINKED ISSUE #5626: Dark theme group name highlight is hard to read
##### Brief Description

Commit hash:  ec500b667306e913cd37d14b6aa1ef8bddfa4d91

Reproducible: Always

##### Steps to reproduce

1. Enter group.
2. Have someone mention your name.

##### Observed Behavior
Highlight colour makes text within the highlight difficult to read:
image

##### Expected Behavior
Highlight or text colour changed for dark theme to make it relatively legible.

### LINKED ISSUE #5629: Dark theme white padding issue
So one of the bits of padding/whitespace on the left side of the chat area seems to be left white on the dark theme, as if to be used as some kind of really thick "divider." It's awfully striking and out of place, and also it makes the GUI a bit inconsistent. For example, look how this space functions on the default theme:
darktheme1

The horizontal divider that separates the chat with the name, status, avatar, etc is evenly padded on both sides with white space. Also, the background of the selected user you're chatting matches the chat background and they are "conjoined" together which looks nice, as the selected user and the chat window all flow together in the same color to make it clear which friend you're chatting. Now look at how the dark theme looks:
darktheme2

That padding is way too thick to be used as some kind of "divider" and even so, a divider doesn't look good as it doesn't keep the style where the selected user and chat background conjoin/flow together. After several painstaking hours in photoshop, I bring you how the dark theme should look:
darktheme3

@TriKriSta

### PR DESCRIPTION
**Pull request by TriKriSta**
add custom themes in AboutFriendForm
edit style for QSpinBox disable
edit palette color for dark theme and etc
edit style for QSplitter

- [ ] Commits follow our [git commit guidelines](https://github.com/qTox/qTox/blob/master/CONTRIBUTING.md#git-commit-guidelines)

Screenshot_20190416_185940
Screenshot_20190416_190200
Screenshot_20190416_190049
Screenshot_20190416_190402
Screenshot_20190419_011020

---
This change is [Reviewable](https://reviewable.io/reviews/qtox/qtox/5627)

### CODE REVIEW

File: src/widget/about/aboutfriendform.cpp [hunk 1]
```diff
@@ -2,6 +2,7 @@
 #include "src/widget/gui.h"
 #include "ui_aboutfriendform.h"
 #include "src/core/core.h"
+#include "../style.h"
```
Comment (Diadlo) [inline src/widget/about/aboutfriendform.cpp]: Please, use full path, if file is not in the same directory. I.e. `#include "src/widget/style.h"`

**Review by sudden6 — APPROVED** [review]
:lgtm_strong:

> Reviewed 4 of 8 files at r1, 7 of 7 files at r2.
> *[Reviewable](https://reviewable.io/reviews/qtox/qtox/5627#-:-LcwdjlWCmkfF5R47H_H:b-jbsf8z)* status: :shipit: complete! 1 of 1 LGTMs obtained

**Comment by sudden6** [comment]
Thank you very much again!

**Comment by TriKriSta** [comment]
You are welcome)

**Pull request merged**